.circleci/config.yml

```diff
@@ -437,3 +462,10 @@ workflows:
       - upload_coverage:
           requires:
             - test_cover
+      - reproducible_builds:
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
I think here we should add release branches

Reply:
Let's see how things go while building on master. We shall complete this work once the split of the repo is completed